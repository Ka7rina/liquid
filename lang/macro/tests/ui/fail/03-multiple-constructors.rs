use liquid_lang as liquid;

#[liquid::contract(version = "0.1.0")]
mod noop {
    #[liquid(storage)]
    struct Noop {}

    impl Noop {
        #[liquid(constructor)]
        fn init_0(&mut self) {}

        #[liquid(constructor)]
        fn init_1(&mut self) {}
    }
}

fn main() {}
