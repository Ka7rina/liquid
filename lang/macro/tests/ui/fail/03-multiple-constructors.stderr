error: duplicate constructor definition found here
  --> $DIR/03-multiple-constructors.rs:12:9
   |
12 | /         #[liquid(constructor)]
13 | |         fn init_1(&mut self) {}
   | |_______________________________^
